cli/src/test/java/com/devonfw/tools/ide/context/IdeTestContext.java

```diff
@@ -87,4 +102,85 @@ public IdeTestLogger getLogger() {
 
     return logger;
   }
+
+  /**
+   * Reads the content of the given file and replaces the placeholder "${testbaseurl}" with the actual base URL. Copy from AbstractUrlUpdaterTest.
+   *
+   * @param file the {@link Path} to the file to read.
+   * @param wmRuntimeInfo the {@link WireMockRuntimeInfo} providing the base URL.
+   * @return the resolved file content.
+   */
+  public static String readAndResolve(Path file, WireMockRuntimeInfo wmRuntimeInfo) {
```

Comment:
name more specific like `readAndResolveBaseUrl`?

Reply:
I've renamed it now.